SPECS/libxml++/libxml++.spec

```diff
@@ -0,0 +1,70 @@
+Summary:        libxml++
```

Comment:
Please also update `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md ` for both packages.